modules/administration-guide/pages/extensions-for-microsoft-visual-studio-code-open-source.adoc

```diff
@@ -7,9 +7,27 @@
 [id="extensions-for-microsoft-visual-studio-code-open-source"]
 = Extensions for Microsoft Visual Studio Code - Open Source
 
-A paragraph or more of conceptual information about the embedded link:https://open-vsx.org/[Open VSX Registry] that Che admins need to understand.
+To manage Visual Studio Code extensions {prod-short} uses link:https://open-vsx.org/[Open VSX Registry], it is an open source VS Code extensions registry.
 
-Explain the process and UX implications for Che end-users.
+The default Open VSX registry differs depending on how proxies or DNS is configued. 
+
+If open-vsx.org can be resolved, the default will be to the public link:https://open-vsx.org/[Open VSX Registry]; if not, then the default will be a locally cached Open VSX registry instance within the plugin-registry pod to support airgapped, offline, and proxy-restricted deployments. 
+
+It is also possible to control what instance of Open VSX should be used in the workspaces, should you want to include your own custom build. This configuration is part of the `CheCluster` Custom Resource:
+
+[source,yaml,subs="+quotes"]
+----
+spec:
+  components:
+    pluginRegistry:
+      openVSXURL: "__<Open_VSX_registry_URL>__" # <1> 
+----
+
+<1> Sets the URL of Open VSX registry. The default value is `https://open-vsx.org`. An empty string will cause  {prod-short} to use the local Open VSX registry in the plug-in registry pod.
+
+The local Open VSX registry runs in the plug-in registry pod and provides a subset of the VS Code extensions published on link:https://open-vsx.org/[Open VSX Registry].
+
+.Below, please find information about managing a local Open VSX registry:
```

Comment:
rather than a suggestion, "please find" could this be a section title called "managing a local Open VSX registry" ?